dap_aria_mapping/getters/validation.py

```diff
@@ -0,0 +1,37 @@
+from nesta_ds_utils.loading_saving.S3 import download_obj
+from dap_aria_mapping import BUCKET_NAME
+
+def get_tree_depths() -> dict:
+    """gets a dictionary with metrics about the tree depth of the taxonomies
+
+    Returns:
+        dict: size and depth metrics of trees in taxonomy
+    """
+    return download_obj(
+        BUCKET_NAME,
+        'outputs/validation_metrics/taxonomy_depth.json',
```

Comment:
Similar to comments in the other validation PR, maybe we should adjust these so that you can select a specific taxonomy (with str arguments and f-statements)? While we're hopefully deciding soon on a final taxonomy, part of this decision will be made based on validation results, including entropy measures.

Reply:
Unless I'm getting this wrong (based on today's comment) and in fact there are multiple keys corresponding to each of our taxonomies.